drivers/dac/ltc2688/ltc2688.c

```diff
@@ -469,6 +469,8 @@ int32_t ltc2688_init(struct ltc2688_dev **device,
 		printf("LTC2688 successfully initialized\n");
 	else
 		printf("LTC2688 initialization error (%d)\n", ret);
+	
```

Comment:
can you do?

Reply:
This way the message is printed at each error.

Thanks for the fix.

Reply:
Sure will do this.